src/components/elements/annotationSidePanel/components/AnnotationCard.tsx

```diff
@@ -0,0 +1,95 @@
+/* Import Dependencies */
+import classNames from 'classnames';
+import moment from 'moment';
```

Comment:
this library is depreciated https://terodox.tech/migrating-away-from-momentjs-part1/

Reply:
https://naturalis.atlassian.net/browse/DD-1345?atlOrigin=eyJpIjoiNzEwNjJiNDQzZGQxNDcxOGI3MjEwYmI4YTE5MWZlZjMiLCJwIjoiaiJ9

Reply:
:fire: